ykrt/src/compile/jitc_yk/aot_ir.rs

```diff
@@ -190,6 +261,43 @@ pub(crate) struct Instruction {
     name: RefCell<Option<String>>,
 }
 
+impl PartialEq for Instruction {
+    fn eq(&self, other: &Self) -> bool {
```

Comment:
I'm fine with using pointer equality but exposing it as `=` makes me a bit queasy because `=` is generally assumed to be a weaker condition than pointer equality. Maybe we should just expose a `ptr_eq` function to callers? If we ever need to change the pointer equality-ness, this will make very clear to us every position which needs changing.

Reply:
Fixed in 9d1a0ff18619b4cdb5e42ceed710f2043369c8b4